bubblemans/2021-spring-cs160-team-EduAll
# Student and Professor Registration Pages

**Features completed:** 

1. Student Registration page: Year, Major, Classes Registered For
2. Professor Registration page: Year, Department

**Steps to test:** 
1. Npm start to run the project
2. Go to /register/student
3. Go to /register/professor


Expected Result: Registration pages for both of them 


![Screen Shot 2021-04-08 at 11 53 34 PM](https://user-images.githubusercontent.com/54693397/114140880-b229df80-98c5-11eb-9870-0f111618bdad.png)

![Screen Shot 2021-04-08 at 11 53 48 PM](https://user-images.githubusercontent.com/54693397/114140894-b6ee9380-98c5-11eb-8b61-d18324f70607.png)
